package.json

```diff
@@ -29,6 +29,7 @@
   "devDependencies": {
     "@babel/core": "^7.8.7",
     "@babel/plugin-transform-runtime": "^7.8.3",
+    "@babel/preset-env": "^7.10.2",
```

Comment:
Note that this is required in the `.babelrc` - I think this is why things were failing to build in travis, you may've had it installed locally as a leftover.

Reply:
Ah, okay, good catch.